libs/components/indicators/src/lib/modules/icon/icon-stack.component.scss

```diff
@@ -36,6 +36,10 @@
   }
 }
 
+.sky-icon-inverse {
+  color: var(--sky-top-icon-override, $sky-text-color-on-dark);
+}
+
```

Comment:
outside of the alert, label, toast and status indicator components the first variable `--sky-top-icon-override` is not set so the top icon color is the default text color of #fff. In those components where the variable is set, the icon color is set to the first variable value. customers cannot use edit icon stack directly so this method was used instead of an input